traveler-swift-core/TravelerKit/Extensions/Order+Equatable.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Order+Equatable.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-06-09.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+extension Order: Equatable {
+    public static func == (lhs: Order, rhs: Order) -> Bool {
+        return lhs.id == rhs.id
+    }
```

Comment:
This is not wrong, but I would add this to the model itself..The class is defined in the same module, not need for extensions to live in a separate file